public/js/reducers/sources.js

```diff
@@ -156,24 +171,30 @@ function updateTabList(state, source, tabIndex) {
  * @memberof reducers/sources
  * @static
  */
-function getNewSelectedSourceId(state, id) : ?Source {
+function getNewSelectedSourceId(state, id: any) : ?Source {
```

Comment:
hmm, perhaps you want to always pass an array of `tabIDs` that way we have a consistent API.